### DISCUSSION TITLE
Is there an easy way to get rid of markdown code tags?

### DISCUSSION THREAD

**Discussion by IPWright83** [yaroslavyaroslav/OpenAI-sublime-text#52] [Q&A]
Firstly thanks for creating this plugin, I'm just trying it out for the first time. I'm trying to use Ollama with the `deepseek-coder-v2` model and I'm having difficulty getting a replace command working.

I'm basically trying to get it to replace some JavaScript, but no matter what I tell it to do, it always creates tags around the output:

![image](https://github.com/yaroslavyaroslav/OpenAI-sublime-text/assets/1374775/335c7a02-542d-4980-82ca-d7f6abedea29)

With a prompt of "Create some JSDoc for this code" and the following `assistant_role` `"You are a senior software engineer. Modify the provided text or code based on the prompt you have been given. If the input is code then you should generate an output that does not include markdown syntax with backticks. Focus on generating plain and simple code that is compatible with the language selected. You are strictly forced to skip any changes in such that would violate any of these rules, you're strictly forbidden to wrap response into something and to provide any explanation. Do not include ``` in the output.",`

Do you have any tips on stopping this from happening?

**Comment by yaroslavyaroslav** [comment]
Hey, thanks for the kind words. Few things here. 
1. The main use case for the plugin that I design is to be assistant i.e. a chat in a separate tab/panel at the first place and I really hope you'd give it a try. Thus direct code manipulation is quite under invested here so far. I have a plans to make them better someday in a forceable future, but no promises.
2. In sake of a beautifying the chat history and to provide more context to the model each selected chunk that is being passed to a model is wrapped into the general scope provided by Sublime Text. so under the hood you're passing to the model \`\`\`python your code \`\`\`, that somewhat forces model to answering in the same format.
3. I was able to force GPT-4o model to prompt plain code (without the ```) just yet within append mode, by passing the restrictions that you've tried in your `assistant_role` prompt in the command input panel (the one that appears below on whatever action call) and it worked, but yet it's still quite unstable and annoying.

**Reply by IPWright83** [reply]
Thanks @yaroslavyaroslav , yeah I've read a few posts suggesting that providing negative instructions in this manner to the model is often unreliable, and I was finding I'd get inconsistent results.

I do plan on giving out the chat panel more of a try too!
